Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -20,6 +22,41 @@ final class SettingsViewModel: ObservableObject {
         self.authenticationViewModel = authenticationViewModel
     }
     
+    let urls: [String: URL] = [
+        "mikaelacaronProfile": URL(string: "https://github.com/mikaelacaron")!,
+        "Basic-Car-MaintenanceRepo": URL(string: "https://github.com/mikaelacaron/Basic-Car-Maintenance")!,
+        "bugReport": URL(string: "https://github.com/mikaelacaron/Basic-Car-Maintenance/issues")!
```

Comment:
Don't include this "bug report" section, this issue is only about showing contributors